executables/collector/src/main/java/de/cyface/collector/verticle/CollectorApiVerticle.java

```diff
@@ -304,8 +304,8 @@ private void addAuthenticatedPostV2Handler(final Router router, final JWTAuth jw
         final var jwtAuthHandler = JWTAuthHandler.create(jwtAuth);
         router.post(endpoint)
                 .consumes("multipart/form-data")
-                .handler(bodyHandler)
                 .handler(LoggerHandler.create())
+                .handler(bodyHandler)
```

Comment:
see below, I'm not 100% sure we may call the LoggerHandler before, but I think it's ok.

Reply:
It is enforced by Vert.x in this version

Reply:
Ah, I thought so. Thanks for clarification.